builder/src/build/patches/investigated/update-webpack-chunks-file/index.ts

```diff
@@ -0,0 +1,32 @@
+import { readdirSync, readFileSync, writeFileSync } from "node:fs";
+
+import { NextjsAppPaths } from "../../../../nextjs-paths";
+import { getUpdatedWebpackChunksFileContent } from "./get-updated-webpack-chunks-file-content";
+
+/**
+ * Fixes the webpack-runtime.js file by removing its webpack dynamic requires.
+ *
+ * This hack is especially bad for two reasons:
+ *  - it requires setting `experimental.serverMinification` to `false` in the app's config file
```

Comment:
Is this still true?

Reply:
nope! 😅 👍

Reply:
I've update the comment, the first point still stands unfortunatenly 🥲 

I've created a [new issue](https://github.com/flarelabs-net/poc-next/issues/14) to track the second point (I had this issue in my original repo)